recipe/meta.yaml

```diff
@@ -24,15 +21,74 @@ requirements:
     - zlib
     - ncurses
     - readline
-  run:
-    - python
-    - zlib
-    - ncurses
-    - readline
 
-test:
-  commands:
-    - test -f "${PREFIX}/lib/libmount.so"
+outputs:
+  - name: util-linux-libs
+    script: install-util-linux-libs.sh
+    build:
+      run_exports:
+        - {{ pin_subpackage('util-linux-libs', max_pin='x.x') }}
+    requirements:
+      build:
+        - make
+      host:
+      run:
+      run_constrained:
+        # TODO: Either split util-linux-libs into their separate libraries
+        #       or decide whether to use libuuid from util-linux or e2fsprogs
+        #       for the "libuuid" package. Also beware of macOS issue
+        #       https://github.com/conda-forge/libuuid-feedstock/issues/16 .
+        # The "libuuid" package also contains libuuid built from util-linux.
```

Comment:
As indicated by this comment, I've also pondered if we'd have to split this even further...
I think for now it makes sense to go with the simple split with `util-linux-libs` holding all of the shared libraries.
(BTW, the `util-linux-libs` naming is also what is now used by Arch Linux and I believe it is the one that makes the most sense.)